packages/vue-renderer/src/renderer.js

```diff
@@ -103,10 +103,16 @@ export default class VueRenderer {
       await this.loadResources(fs)
 
       // Verify
-      if (!this.isReady && this.context.options._start) {
-        throw new Error(
-          'No build files found. Use either `nuxt build` or `builder.build()` or start nuxt in development mode.'
-        )
+      if (this.context.options._start) {
+        if (!this.isReady) {
+          throw new Error(
+            'No build files found. Use either `nuxt build` or `builder.build()` or start nuxt in development mode.'
+          )
+        } else if (this.context.options.modern && !this.context.resources.modernManifest) {
```

Comment:
Shouldn't we also include this check for isReady?

Reply:
I think so too

Reply:
@pi0 @Atinux Since modern needs a different error message, so I didn't add it into isReady, do you mean change isReady to not only return boolean ?

Reply:
> do you mean change isReady to not only return boolean

I like this idea